src/cf-notification/src/cf-notification.less

```diff
@@ -0,0 +1,44 @@
+/* ==========================================================================
+   Capital Framework
+   Notifications Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// Color variables
+
+@notification-bg:             #d2d3d5;
```

Comment:
I'm noticing a pattern where we have the atomic prefix on the class names, but not on variables. I am fine with continuing this pattern.